flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -1085,6 +1085,54 @@ public void setUseSnapshotCompression(boolean useSnapshotCompression) {
         configuration.set(ExecutionOptions.SNAPSHOT_COMPRESSION, useSnapshotCompression);
     }
 
+    // --------------------------------------------------------------------------------------------
+    //  Asynchronous execution configurations
+    // --------------------------------------------------------------------------------------------
+
+    @Internal
```

Comment:
It might be better to mark them as `@Experimental` instead of `@Internal`.

- Internal: Annotation to mark methods within **stable**, public APIs as an internal developer API.
- Experimental: Classes with this annotation are neither battle-tested **nor stable**, and may be changed or removed in future versions.

Reply:
Seems that this comment is not resolved yet?

Reply:
Ah, I misread the comment, and changed them from `@Experimental` to `@Internal`, I have corrected them now.